engines/glk/archetype/id_table.cpp

```diff
@@ -47,16 +45,16 @@ int add_ident(const String &id_str) {
 
 	if (p->next == nullptr || *p->next->id_name > id_str) {
 		new_rec = new IdRecType();
-		append_to_xarray(h_index, new_rec);
+		append_to_xarray(g_vm->h_index, new_rec);
```

Comment:
The GitHub Actions builds seem to be failing because of a missing include in this file.